blog/01-18-2023-pros-and-cons-of-celo-api/pros-and-cons-of-celo-api.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Pros and cons of using Celo API to create Defi applications
```

Comment:
I advise getting rid of API in the title and the description since in the main documentation, the APIs are not part of the discussion

Reply:
okay